graphql/mutation_resolver.go

```diff
@@ -0,0 +1,1133 @@
+package graphql
```

Comment:
@khelif96 do we foresee ever wanting to split mututations up into more files? (Could be a future ticket to think about the best way to do it but does seem like this file is also quite large)

Reply:
mutations are tougher to split up since they all stem from the mutation root. and are harder to associate with parent resolvers. [Federated Graphql ftw](https://www.apollographql.com/docs/federation/) But for now this might be the best ew can do.